las_geoh5/import_las.py

```diff
@@ -0,0 +1,219 @@
+#  Copyright (c) 2023 Mira Geoscience Ltd.
+#
+#  This file is part of my_app project.
+#
+#  All rights reserved.
+#
+from __future__ import annotations
+
+import warnings
+from pathlib import Path
+
+import lasio
+import numpy as np
+from geoh5py import Workspace
+from geoh5py.groups import DrillholeGroup
+from geoh5py.objects import Drillhole
+from geoh5py.shared.concatenation import ConcatenatedDrillhole
+
+
+def get_depths(lasfile: lasio.LASFile):
+    """
+    Get depth data from las file.
+
+    :param lasfile: Las file object.
+
+    :return: Depth data.

```

Comment:
is there a specific type for the returned data? (numpy, array?)

Reply:
There is now!